File: auto-docs/executables/julia/2d_histogram.jl
# Learn about API authentication here: plot.ly/julia/getting-started
# Find your api_key here: plot.ly/settings/api

using Plotly

x = randn(500)
y = randn(500)+1

Plotly.signin("TestBot", "r1neazxo9w")
data = [
  [
    "x" => x, 
    "y" => y, 
    "type" => "histogram2d"
  ]
]
response = Plotly.plot(data, ["filename" => "2d-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]

File: auto-docs/executables/python/2d_histogram.py
# Learn about API authentication here: {{BASE_URL}}/python/getting-started
# Find your api_key here: {{BASE_URL}}/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

import numpy as np

x = np.random.randn(500)
y = np.random.randn(500)+1

py.sign_in('TestBot', 'r1neazxo9w')
data = Data([
    Histogram2d(
        x=x,
        y=y
    )
])
plot_url = py.plot(data, filename='2d-histogram', auto_open=False)
